PocketCastsTests/Tests/IAP/IAPHelperTests.swift

```diff
@@ -0,0 +1,87 @@
+import XCTest
+import StoreKitTest
+
+@testable import podcasts
+import PocketCastsServer
+
+final class IAPHelperTests: XCTestCase {
+
+    var email: String?
+
+    override func setUpWithError() throws {
+        // Pretend we're logged in
+        email = ServerSettings.syncingEmail()
+        ServerSettings.setSyncingEmail(email: "test@test.com")
+    }
+
+    override func tearDownWithError() throws {
+        // Restore previous login
+        ServerSettings.setSyncingEmail(email: email)
+    }
```

Comment:
[nit] I wonder if we can move this to a `LoggedInXCTestCase` and just inherit from it?

Reply:
I will do this in a separate PR.